### PR TITLE
Condense Deploy widget for old Hygieia UI and syntax 

### PR DESCRIPTION
**Pull request by Courtneyp123**
Users have the option to choose between seeing a max of 5 listings or 10 listings with buttons.  I also removed extra comments and spaces throughout my code from last PR and my new methods.

### CODE REVIEW

File: UI/src/components/widgets/deploy/view.js [hunk 1]
```diff
@@ -129,7 +137,17 @@
 
         function environmentsCallback(data) {
             //$scope.$apply(function () {
```
Comment (Sbrenthughes) [inline UI/src/components/widgets/deploy/view.js]: Remove commented line here

File: UI/src/components/widgets/deploy/view.js [hunk 2]
```diff
@@ -129,7 +137,17 @@
...
+                // Collect last 5 or 10 listings in deploy widget
+                if(length > ctrl.listingNum){
+                    for(var i = length - ctrl.listingNum; i <= length -1; i ++){
+                        dataList.push(data.environments[i]);
+                    }
+                    ctrl.environments = dataList;
+                } else{
+                    ctrl.environments = data.environments;
+                }
             //});
```
Comment (Sbrenthughes) [inline UI/src/components/widgets/deploy/view.js]: Remove commented line

**Comment by Courtneyp123** [comment]
Changes added for Deploy widget and I removed comments.  @rvema @Sbrenthughes

File: UI/src/components/widgets/deploy/view.html [hunk 3]
```diff
@@ -1,5 +1,28 @@
 <div class="deploy-view">
     <div hide-empty="['{{ deployView.environments[0] }}']">
+        <style>
```
Comment (Sbrenthughes) [inline UI/src/components/widgets/deploy/view.html]: This needs to go into it's on file. Create a `deploy.less` You can follow the example from the build widget

File: UI/src/components/widgets/deploy/view.html [hunk 4]
```diff
@@ -25,5 +48,18 @@
                 </td>
             </tr>
         </table>
+
+        <script>
+            function openPage(pageName,elmnt,color) {
```
Comment (Sbrenthughes) [inline UI/src/components/widgets/deploy/view.html]: This looks strange. Let's pull up tomorrow to discuss.

File: UI/src/components/widgets/deploy/view.js [hunk 5]
```diff
@@ -128,9 +142,17 @@
...
 
         function environmentsCallback(data) {
-            //$scope.$apply(function () {
-                ctrl.environments = data.environments;
-            //});
+                var dataList = [];
+                ctrl.totalListingCount = data.environments.length;
+                // Collect last 5 or 10 listings in deploy widget
+                if(ctrl.totalListingCount > ctrl.listingNum){
+                    for(var i = ctrl.totalListingCount - ctrl.listingNum; i <= ctrl.totalListingCount -1; i ++){
```
Comment (Sbrenthughes) [inline UI/src/components/widgets/deploy/view.js]: Let's discuss this further as well

File: UI/src/components/widgets/deploy/view.html (hunk 3 above)
Comment (Courtneyp123) [reply UI/src/components/widgets/deploy/view.html]: On it!

**Comment by Courtneyp123** [comment]
@rvema @Sbrenthughes Changes made for tab view.  Stephen helped me make code more efficient and improve tab view.  

I checked and confirmed that tabs are collecting last 5 and 10 listings correctly.  No error is thrown if the list is empty.

File: UI/src/components/widgets/deploy/view.js [hunk 6]
```diff
@@ -14,12 +14,26 @@
         ctrl.environments = [];
         ctrl.statuses = DashStatus;
         ctrl.ignoreEnvironmentFailuresRegex=/^$/;
+        ctrl.listingNum = 5;
```
Comment (Sbrenthughes) [inline UI/src/components/widgets/deploy/view.js]: If this var isn’t used anymore let’s remove it.

**Pull request merged**